### ISSUE TITLE
Update badge specs on Components page

### ISSUE THREAD

**Issue by santokii** [google/material-design-lite#444]
Simplify badge options to 4 (see comp):

Icon with number badge
Icon with icon badge
Text with number badge
Text with icon badge

For accessibility reasons the smallest font size we should be using for Roboto is 12, so lets increase the font size from 10 to 12, and increase the size of the circle from 20 to 22

For legibility lets increase the font-weight from 400 to 600

To put more focus on the badge (not the example icon) lets use the same icon and decrease opacity to 24% https://www.gstatic.com/images/icons/material/system/1x/account_box_black_24dp.png

Badge should sit 10px in the upper right corner of the icon.

Icon in the badge should be 12x12 (or 16x16 with bounding box)

Always vertically and horizontally center numbers/icons in the badge)

Text badges should sit 4 pixels up and to the right (from the exact bottom of the text)

![02 components - badges cropped](https://cloud.githubusercontent.com/assets/11649557/8327294/2c793984-1a5f-11e5-8934-db8fc7bbddbf.png)
